src/components/Steps/index.js

```diff
@@ -9,7 +9,7 @@ import { stepsPropTypes } from './proptypes';
 const mapSteps = (steps, singleStep) =>
   // singleStep boolean param to determine whether to render steps in a list (ul) or a single item for StepWithOptions and StepBasic
   steps.map((step, index) => {
-    if (step.stepType === 'step_with_options_accordian')
+    if (step.type === 'step_with_options_accordian')
```

Comment:
just curious, when/where did this change?

Reply:
Its affected on the joplin side, since essentially its just passing on the name of the model field